toolkit/Makefile

```diff
@@ -90,6 +89,7 @@ IMAGEGEN_DIR       ?= $(BUILD_DIR)/imagegen
 TOOLCHAIN_RPMS_DIR ?= $(BUILD_DIR)/toolchain_rpms
 TIMESTAMP_DIR      ?= $(BUILD_DIR)/timestamp
 PROFILE_DIR        ?= $(BUILD_DIR)/profile
+CCACHE_DIR         ?= $(BUILD_DIR)/ccache
```

Comment:
I think this would be a breaking change for an existing build environment (at least, it would create a new ccache folder. and it changes the behavior so that "make clean" will now clean the ccache folder, where it did not before